docs/DeveloperGuide.adoc

```diff
@@ -767,6 +845,79 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+=== Use case: Sync Account
+
+*MSS*
+
+1.  User requests to sync account
+2.  TunedIn asks for Facebook ID
+3.  User confirms that the account is of the user's
+4.  TunedIn pulls the data from the Facebook ID using the Graph API
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. The Facebook ID does not exist
++
+Use case ends.
+
+* 3a. The given Facebook ID is inaccurate
++
+[none]
+** 3a1. TunedIn shows an error message.
++
+Use case resumes at step 2.
+
+=== Use case: Logging in
+
+*MSS*
+
+1.  User types in password
+2.  TunedIn checks password accuracy
+3.  TunedIn brings the patient to the home page
++
```

Comment:
Take note that use case generally follows a `User -> System -> User -> System`.

Reply:
Will do, thank you.